apps/condo/domains/notification/schema/Message.js

```diff
@@ -106,6 +106,13 @@ const Message = new GQLListSchema('Message', {
             type: 'DateTimeUtc',
             isRequired: false,
         },
+
```

Comment:
you need to rebase it's not a part of your PR

Reply:
I get rid of this. Rebased